modules/tenant-management/src/Volo.Abp.TenantManagement.Domain.Shared/Volo/Abp/TenantManagement/Localization/Resources/en.json

```diff
@@ -15,6 +15,9 @@
     "Permission:Edit": "Edit",
     "Permission:Delete": "Delete",
     "Permission:ManageConnectionStrings": "Manage connection strings",
-    "Permission:ManageFeatures": "Manage features"
+    "Permission:ManageFeatures": "Manage features",
+    "DisplayName:AdminEmailAddress": "Admin Email Address",
+    "DisplayName:AdminPassword": "Admin Password",
+    "InvalidAdminProperties": "Admin email address or password is invalid."
```

Comment:
Remove custom message (will not needed when you remove the Validate method above)